src/utils/graph/layout.js

```diff
@@ -16,7 +16,7 @@ import {
  * @param {array} params.nodes The input nodes
  * @param {array} params.edges The input edges
  * @param {object=} params.layers The node layers if specified
- * @param {number} params.basisX The basis relating diagram width in X
+ * @param {number} params.spreadX The amount to adapt spacing in X
```

Comment:
I wonder what is the main motivation behind adapting spacing instead of the relation to the width of the diagram, and how does it improve the layout engine? It would be worth mentioning which goal this change relates to in the PR decription.

Reply:
Ah I've updated the comment here to make it a bit clearer what that parameter does, though it's not really expected that this value will change from the included default constant which is pretty implementation relative and was found by trial and error.

If you're really curious though it's worth following the parameter through the code and playing with the value while watching in the browser.